operations/README.md

```diff
@@ -369,6 +370,7 @@ Mechanisms used by the system to ensure the security and authentication of inter
 - [API Authorization](security_and_authentication/API_Authorization.md)
 - [Retrieve an Authentication Token](security_and_authentication/Retrieve_an_Authentication_Token.md)
 - [Manage Sealed Secrets](security_and_authentication/Manage_Sealed_Secrets.md)
+- [SecureOperations](security_and_authentication/SOPS.md)
```

Comment:
Should this match the level-1 header? Otherwise, please fix the spellcheck.
```suggestion
- [SOPS Introduction](security_and_authentication/SOPS.md)
```